packages/cyberstorm/src/components/Layout/Developers/PackageUpload/PackageUploadLayout.tsx

```diff
@@ -11,11 +11,13 @@ import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { BaseLayout } from "../../BaseLayout/BaseLayout";
 import { CreateTeamForm } from "@thunderstore/cyberstorm-forms";
 import { Dialog } from "../../../..";
+import { useState } from "react";
 
 /**
  * Cyberstorm PackageUpload Layout
  */
 export function PackageUploadLayout() {
+  const [dialogOpen, setOpenDialog] = useState(false);
```

Comment:
Just mentioning this once more since I see no reason to not adhere to the naming convention of `[x, setX]` here.

Reply:
Yeah, weird, seems like I keep making this mistake. Probably renaming just the variable and then forgetting the setter